efficientdet/keras/util_keras.py

```diff
@@ -146,10 +146,20 @@ def restore_ckpt(model,
   if tf.io.gfile.isdir(ckpt_path_or_file):
     ckpt_path_or_file = tf.train.latest_checkpoint(ckpt_path_or_file)
 
-  if (tf.train.list_variables(ckpt_path_or_file)[0][0] ==
+  var_list = tf.train.list_variables(ckpt_path_or_file)
+  if (var_list[0][0] ==
       '_CHECKPOINTABLE_OBJECT_GRAPH'):
     try:
-      model.load_weights(ckpt_path_or_file)
+      # Use custom checkpoint solves mismatch shape issue.
+      keys = {var[0].split('/')[0] for var in var_list}
+      keys.discard('_CHECKPOINTABLE_OBJECT_GRAPH')
+      if skip_mismatch:
```

Comment:
"skip_mismatch" should cover all variables that have shape mismatched? (not only class_net)?

Reply:
The current Checkpoint approach doesn't support shape mismatch. Maybe I should add exclude vars back?